.gitignore

```diff
@@ -2,6 +2,7 @@
 
 # Packages
 *.egg
+*.eggs
```

Comment:
I was running a few build commands using `setup.py` and noticed that it created this directory.

It must be something to do with `build` or `dist` in `setup.py`